content/dev/dev-guide.md

```diff
@@ -0,0 +1,325 @@
+   - ```bash
+     ./urbit -F zod -B ~/urbit-git/bin/multi.pill
+     ```
+ - When it's finished you'll see a command prompt like `~zod:dojo>` - this is the urbit dojo where you can enter and run urbit commands.
+ - To exit the dojo type `|exit` or `ctrl-D`
+
+#### Fake ship networking
+
+ - Fake ships on the same machine can automatically talk to each other. Let's create a second fake ship named `~bus` and confirm this.
+   - ```bash
+     ./urbit -F bus -B ~/urbit-git/bin/multi.pill
+     ```
+ - Now you'll have two local ships, `~zod` and `~bus`.
+ - From the newly created `~bus` dojo enter `|hi ~zod` and you'll see that urbit's networking finds the first fake ship we created.
+ - Exit the `~bus` dojo with `|exit` or `ctrl-D`
+
+### Desks
+
+#### What is a desk?
+
+A desk is an independently revision-controlled branch of a ship that uses the Clay filesystem. Each desk contains its own apps, mark definitions, files, etc. You can think of a desk like a git branch you're working in.
+
+A basic ship has a `%base`, `%garden`, `%landscape`, `%bitcoin`, and `webterm` desk (we mounted these earlier). The `%base` desk has the kernel and base system software, while other desks are each for different apps. 
+
+A desk is a series of numbered commits, the most recent of which represents the current state of the desk. A commit is composed of an absolute time when it was created, a list of zero or more parents, and a map from paths to data.
+
+For building applications to distribute on urbit, we'll want to work on a new desk.
+
+#### Desk Basics
+
+##### Mounting and Unmounting Desks
+
+ - Mounting a desk allows you to add files to it (it creates a directory you can see on the host filesystem). Run these commands in the dojo.
+   - ```bash
+     |mount %base
+     ```
+ - You can similarly unmount a desk
+   - ```bash
+     |unmount %base
+     ```
+
+##### Setting Up a New Desk
+
+ - To create a new desk we'll need to merge from an existing one, think of this like creating a new branch.
+ - Let's create a new `hello` desk by merging from `%garden` (`garden` is just an old name for the application now referred to as `landscape`).
+ - **Note**: Thinking about dependencies:
+   - When we merge from `%garden` we start with its base files in our application.
+   - Since we're copying these files over, its possible an API change could happen in `urbit-git` while we're building our app.
+   - It's good practice to run the merge command at the start of development and prior to deployment (after a fresh pull of `urbit-git`) to make sure it works with the up to date system changes.
+  - Let's start up our fake zod.
+    - ```bash
+      cd ~/urbit-dev
+      ./urbit zod
+      ```
+  - Create our desk in the dojo by merging from `%garden`
+    - ```bash
+      |merge %hello our %garden
+      ```
+  - Mount `%hello` so we can access it from the host filesystem.
+    - ```bash
+      |mount %hello
+      ```
+  - Exit the Dojo with `|exit` or `ctrl-D` 
```

Comment:
+1 to two windows.

Also consider prefacing Bash commands with `$` and Dojo commands with `dojo>`.

Reply:
I avoid the preface just because it can make copy pasting commands a pain.

Can do two windows, but it increases the risk of them accidentally running the same ship at the same time though (which I guess matters less for this? should also get caught by the lock I think).